gateway/webrtc/client.js

```diff
@@ -227,17 +244,11 @@ loginBtn.addEventListener("click", function(event) {
 function handleVideo(myStream) {
     console.log("Entering handleVideo", myStream);
     stream = myStream;
-
-
-    //displaying local video stream on the page
-    if (window.URL) {
-        localVideo.src = window.URL.createObjectURL(stream);
-    } else {
-        localVideo.src = stream;
-    }
+    localVideo.srcObject = stream;
 
     var configuration = {
         "iceTransportPolicy": "relay",
+        iceTransports: 'all',
```

Comment:
I don't see iceTranports under https://www.w3.org/TR/webrtc/.  ??  Also, please standardize on formatting:

 `iceTransports: 'all'`, IMHO   looks better to me rather than `"iceTransportPolicy": "relay",`